src/catalog/src/local/manager.rs

```diff
@@ -64,18 +65,15 @@ pub struct LocalCatalogManager {
 
 impl LocalCatalogManager {
     /// Create a new [CatalogManager] with given user catalogs and table engine
-    pub async fn try_new(engine: TableEngineRef) -> Result<Self> {
+    pub async fn try_new(engine_manager: TableEngineManagerRef) -> Result<Self> {
+        let engine = engine_manager.default();
```

Comment:
No "default", just get "mito" engine. Return an error if "mito" engine is not found. It's caller's responsibility to provide a correct TableEngineManager.